mesonbuild/compilers/c.py

```diff
@@ -427,7 +427,13 @@ def _get_compiler_check_args(self, env, extra_args, dependencies, mode='compile'
                 args += env.coredata.get_external_preprocess_args(self.language)
             elif mode == 'compile':
                 # Add CFLAGS/CXXFLAGS/OBJCFLAGS/OBJCXXFLAGS from the env
-                args += env.coredata.get_external_args(self.language)
+                sys_args = env.coredata.get_external_args(self.language)
```

Comment:
__[Flake8]__ <sub>posted by Sider</sub>

[F841] local variable 'sys_args' is assigned to but never used